Dockerfile

```diff
@@ -90,7 +106,7 @@ CMD ["/usr/local/bin/kargo"]
 ####################################################################################################
 FROM --platform=$BUILDPLATFORM docker.io/library/node:20.12.2 AS ui-dev
 
-RUN npm install --global pnpm
+RUN npm install --global pnpm@9.0.2
```

Comment:
This is fixed in `main` already, but we fixed it by upgrading the version of pnpm that our `package.json` is compatible with.

Reply:
I think a new version just got released: https://github.com/pnpm/pnpm/releases/tag/v9.0.3